src/webots/nodes/utils/WbSupervisorUtilities.cpp

```diff
@@ -503,7 +503,13 @@ WbNode *WbSupervisorUtilities::getProtoParameterNodeInstance(int nodeId, const Q
     mRobot->warn(tr("%1: node not found.").arg(functionName));
     return NULL;
   }
-  return static_cast<WbBaseNode *>(node)->getFirstFinalizedProtoInstance();
+  WbBaseNode *proto = static_cast<WbBaseNode *>(node)->getFirstFinalizedProtoInstance();
+  if (!proto)
+  {
+    mRobot->warn(tr("Cannot get the PROTO instance for node `%1` (type %2).").arg(node->modelName(), node->nodeModelName()));
```

Comment:
It would be better to use standard single quotes:
```suggestion
    mRobot->warn(tr("Cannot get the PROTO instance for node '%1' (type '%2').").arg(node->modelName(), node->nodeModelName()));
```